lib/std/io/tty.zig

```diff
@@ -59,8 +60,6 @@ pub const Color = enum {
     reset,
 };
 
-/// Provides simple functionality for manipulating the terminal in some way,
-/// such as coloring text, etc.
```

Comment:
This is some useless doc comment I wrote previously that provides no information that isn't already there if you just look at what's in Config and find that it can only set color